io.github.nyre221.kiview.json

```diff
@@ -0,0 +1,45 @@
+{
+   "base": "io.qt.qtwebengine.BaseApp",
+   "base-version": "6.6",
+   "command": "kiview",
+   "build-options": {
+    "env": [
+      "QTWEBENGINE_DISABLE_SANDBOX=1"
+    ]
+  },
+
+   "finish-args": [
+      "--share=ipc",
+      "--device=dri",
+      "--socket=fallback-x11",
+      "--socket=wayland",
+      "--env=QML_IMPORT_PATH=/app/qml" ,
+      "--socket=pulseaudio",
+      "--socket=session-bus",
+      "--filesystem=host:ro",
+      "--filesystem=/tmp",
+      "--talk-name=org.freedesktop.Flatpak",
+      "--talk-name=org.kde.klipper",
+      "--talk-name=org.freedesktop.DBus",
+      "--talk-name=org.kde.dolphin"
+   ],
+   "cleanup-commands":[ "/app/cleanup-BaseApp.sh"],
+   "id": "io.github.nyre221.kiview",
+   "modules": [
+      {
+         "buildsystem": "cmake-ninja",
+         "name": "kiview",
+         "sources": [
+            {
+               "type": "git",
+               "url": "https://github.com/Nyre221/Kiview.git",
+               "commit":"fdc79c3a64f15005d6dca8bf9406c917a5cfa504"
+            }
+         ]
+      }
+
+   ],
+   "runtime": "org.kde.Platform",
+    "runtime-version": "6.6",
+   "sdk": "org.kde.Sdk"
```

Comment:
these three lines should also be toward the top. and please fix the indentation.

Reply:
I'll fix it soon

Reply:
fixed